### ISSUE TITLE
Transition does not work correctly!

### ISSUE THREAD

**Issue by atpcom** [jonataslaw/getx#345]
**Describe the bug**
Black field on the screen when one page changes to another page

**To Reproduce**
1. Go to  first page
2. Go to another page ( authController: ```dart event == null ? Get.offAllNamed('/login') : Get.offAllNamed('/shops'); ```)
3. See black field

**Screenshots**
[Link](https://drive.google.com/file/d/1vFSiUxlzcKPYPVotYzRVdYV-xyV5cCV7/view?usp=sharing)

**Flutter Version:**
1.17.5

**Dart Version:**
2.8.4

**Get Version:**
3.2.1

**Describe on which device you found the bug:**
ex: Google Pixel 3 - Android 10.

**Minimal reproduce code**
```dart
void main() {
  Get.put(AuthController());
  runApp(
    GetMaterialApp(
      translations: Translation(),
      locale: Locale('ru', 'RU'),
      debugShowCheckedModeBanner: false,
      transitionDuration: Duration(seconds: 10),
      getPages: [
        GetPage(name: '/', page: () => SplashPage()),
        GetPage(name: '/login', page: () => LoginPage()),
        GetPage(name: '/shops', page: () => ShopsPage()),
      ],
      initialRoute: '/',
    ),
  );
}

class SplashPage extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Center(
        child: JumpingText('download'.tr),
      ),
    );
  }
}

class LoginPage extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Center(
        child: GetX(
            init: InternetController(),
            builder: (InternetController controller) {
              if (controller.isConnect != null && !controller.isConnect)
                Get.snackbar('attention'.tr, 'connection'.tr,
                    instantInit: false);
              return controller.isConnect == null
                  ? JumpingText('download'.tr)
                  : Center(
                      child: RaisedButton(
                        child: Row(
                          mainAxisAlignment: MainAxisAlignment.spaceBetween,
                          mainAxisSize: MainAxisSize.min,
                          children: <Widget>[
                            Image.asset(
                              'assets/images/google-logo.png',
                              width: 30,
                              height: 30,
                            ),
                            Text('signUp'.tr),
                          ],
                        ),
                        shape: StadiumBorder(),
                        onPressed: controller.isConnect ? () {} : null,
                      ),
                    );
            }),
      ),
    );
  }
}


class AuthController extends GetxController {
  final AuthApi _authApi = AuthApi();
  final _user = Rx<FirebaseUser>();

  FirebaseUser get user => _user.value;

  @override
  void onInit() {
    _authApi.getFirebaseUser().listen((event) {
      _user.value = event;
      event == null ? Get.offAllNamed('/login') : Get.offAllNamed('/shops'); 
    });
    super.onInit();
  }

  signIn() => _authApi.signInWithGoogle();
  signOut() => _authApi.signOut();
}
```

**Comment by jonataslaw** [comment]
New version of Flutter seems to have caused some stability in the transitions.
I will resolve this now.

**Comment by binaris-no** [comment]
Confirmed for iOS simulator with Flutter beta. Even the type of the animation is wrong with get 3.2.1.

**Comment by jonataslaw** [comment]
Fix on 3.2.2
